content/body/meter.php

```diff
@@ -0,0 +1,156 @@
+  For example, meter could be used to display a device's current battery or a car's remaining fuel level. 
+  In cases where there is no fixed maxium, it is preferable to utilize the <a href="/progress.php">progress bar</a> component.
+</p>
+
+<p>
+  This page provides examples of both HTML and aria based solutions for creating accessible meter components.
+</p>
+
+<h2>HTML5 Meter Example</h2>
+<?php includeStats(["isForNewBuilds" => true]); ?>
+
+<p>
+  Use of the HTML5 meter element is supported in most browsers. As defined in the 
+  <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meter#technical_summary">MDN docs</a>, 
+  this will utilize the meter role which will read out the current percentage of the meter.
+</p>
+
+<div id="html5-example" class="enable-example">
+  <strong>Storage Space</strong>
+
+  <div class="container">
+    <label id="disk-c-meter-html5" aria-hidden="true">Disk C:</label>
+    <meter aria-labelledby="disk-c-meter-html5" class="enable-custom-meter" value="0.2" min="0" max="1" optimum="0" low="0.2" high="0.8"></meter>
+  
+    <label id="disk-d-meter-html5" aria-hidden="true">Disk D:</label>
+    <meter aria-labelledby="disk-d-meter-html5" class="enable-custom-meter" value="0.9" min="0" max="1" optimum="0" low="0.2" high="0.8"></meter>
+
+    <label id="disk-e-meter-html5" aria-hidden="true">Disk E:</label>
+    <meter aria-labelledby="disk-e-meter-html5" class="enable-custom-meter" value="0.6" min="0" max="1" optimum="0" low="0.2" high="0.8"></meter>
+  </div>
+</div>
+
+<?php includeShowcode("html5-example"); ?>
+<script type="application/json" id="html5-example-props">
+{
+    "replaceHtmlRules": {},
+    "steps": [{
+            "label": "Use the HTML5 meter tag to create your meter component.",
+            "highlight": "%OPENCLOSECONTENTTAG%meter",
+            "notes": "Set the <code>min</code> and <code>max</code> values to the min and max values of what your meter bar is measuring. <code>value</code> is the current value. Setting the <code>low</code>, <code>high</code>, and <code>optimum</code> fields will result in browser specific colors."
+        },
+        {
+          "label": "Add the <code>aria-label</code> attribute to announce the name of each meter.",
+          "highlight": "aria-label",
+            "notes": ""
+        },
+        {
+            "label": "Apply <code>aria-hidden</code> to any visual labels to prevent redundant items being read out.",
+            "highlight": "%OPENCLOSECONTENTTAG%p",
+            "notes": ""
+        }
+    ]
+}
+</script>
+
+<h2>ARIA role="meter" Example</h2>
+
+<p>
+  Should a case arise where there HTML5 meter component is not accessible for a given browser, the following can be used as an alternative.
+</p>
+
+<p>
+  If you are using VNU for validation, there is a <a href="https://github.com/validator/validator/issues/1380">known issue</a> with <code>role="meter"</code>.
```

Comment:
_nit_
```suggestion
  If you are using v.Nu for validation, there is a <a href="https://github.com/validator/validator/issues/1380">known issue</a> with <code>role="meter"</code>.
```

Also, should we start the line with "Hint: "?